README.md

```diff
@@ -11,14 +11,33 @@ If you work a with a git rebase flow, git machete will (vastly!) help you manage
 
 ## Install
 
-Run the following commands to install git machete:
+Installation could be done by following commands:
 
 ```bash
 $ git clone https://github.com/VirtusLab/git-machete.git
 $ cd git-machete
-$ sudo make install
+$ sudo python setup.py install
 ```
 
+To install you will need python with installed `pip` and `setuptools`.
+
+python could be installed with system packages or using [pyenv](https://github.com/pyenv/pyenv).
+
+Regular python env should have `pip` and `setuptools`.
+
+If python env is missing any of those tools, please execute following commands:
+
+```bash
+# missing pip? check with command: pip
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python get-pip.py
+
+# missing setuptools? check with command: pythom -m "import setuptools"
+pip install setuptools
+```
+
+[More](https://pip.pypa.io/en/stable/installing/) information about `pip` installation.
```

Comment:
Btw I guess this entire procedure could be extracted to a install.sh script... added https://github.com/VirtusLab/git-machete/issues/36 for that

Reply:
IMO no. `python` and `pip` are default things. Installing python project is possible with `setup.py` what is a default way. Like in Java with `mvn` or `bundle` in Ruby.

Reply:
Ok, let's skip then